trello/card.py

```diff
@@ -159,6 +159,7 @@ def from_json(cls, parent, json_obj):
         card.badges = json_obj['badges']
         card.customFields = card.fetch_custom_fields(json_obj=json_obj)
         card.countCheckItems = json_obj['badges']['checkItems']
```

Comment:
card.countCheckItems could probably just be removed, I didn't find any other usages in the code except for the one in fetch_checklists, but I didn't want to involuntarily mess up stuff in other parts so i kept it.